README.md

```diff
@@ -40,6 +40,15 @@ Please use the following to reach members of the community:
 - Twitter: [@rook_io](https://twitter.com/rook_io)
 - Email: [info@rook.io](mailto:info@rook.io)
 
+### Community Meeting
+A regular community meeting takes place every other [Monday at 9:00 AM PT (Pacific Time)](https://zoom.us/j/392602367).
```

Comment:
I think it would be good to add where changes to the date are announced. Currently they are mostly posted (only) to Slack (would be good to overall also announce them on the Google Group?).

Reply:
I'd prefer to not have to open a PR and commit something to the `rook/rook` repo if/when there is a change.  This content should only be static in my opinion.  I do agree about using the google group too, these are the 3 ways that meeting changes would be announced:
1. Slack `#announcements` and/or `#general`
1. Agenda document
1. google group mailing list

Reply:
@jbw976 I don't mean opening a new PR for a change, I just mean adding something like "We announce changes to the meeting date on the following channels: Slack, Agenda, Google Group".

Reply:
Oh I see what you are saying now.  Yes, that suggestion sounds reasonable!